### PR TITLE
Allow Symfony 7

### PR DESCRIPTION
**Pull request by Jean85**
~DO NOT MERGE - on top / targets~ Follows #133

### CODE REVIEW

**Comment by ilario-pierbattista** [comment]
@Jean85 it's ok for you if we merge also #146 before?

**Comment by Jean85** [comment]
Yes! This one just build up onto that

File: src/Capsule/Collection.php [hunk 1]
```diff
@@ -238,7 +234,7 @@ private function notifyQueryExecution(Query $queryLog)
         if (Kernel::VERSION_ID >= 40300) {
             $this->eventDispatcher->dispatch($event, QueryEvent::QUERY_EXECUTED);
         } else {
-            $this->eventDispatcher->dispatch(QueryEvent::QUERY_EXECUTED, $event);
+            $this->eventDispatcher->dispatch($event, QueryEvent::QUERY_EXECUTED);
         }
     }
```
Comment (ilario-pierbattista) [inline src/Capsule/Collection.php:239]: Is this still required?

**Review by ilario-pierbattista — CHANGES_REQUESTED** [review]

**Review by ilario-pierbattista — APPROVED** [review]
LGTM

**Pull request merged**